common/Makefile.common.mk

```diff
@@ -96,15 +96,15 @@ update-common:
 	@git clone -q --depth 1 --single-branch --branch $(UPDATE_BRANCH) https://github.com/istio/common-files $(TMP)/common-files
 	@cd $(TMP)/common-files ; git rev-parse HEAD >files/common/.commonfiles.sha
 	@rm -fr common
-	@cp -ar $(TMP)/common-files/files/* $(shell pwd)
+	@cp -a $(TMP)/common-files/files/* $(shell pwd)
 	@rm -fr $(TMP)/common-files
 
 update-common-protos:
 	@mkdir -p $(TMP)
 	@git clone -q --depth 1 --single-branch --branch $(UPDATE_BRANCH) https://github.com/istio/common-files $(TMP)/common-files
 	@cd $(TMP)/common-files ; git rev-parse HEAD > common-protos/.commonfiles.sha
 	@rm -fr common-protos
-	@cp -ar $(TMP)/common-files/common-protos $(shell pwd)
+	@cp -a $(TMP)/common-files/common-protos $(shell pwd)
```

Comment:
Why remove `-r`?

Reply:
this comes from common-files repo. -a includes -r already.

Reply:
oh right, of course